src/d2_docker/utils.py

```diff
@@ -487,7 +487,7 @@ def export_database(image_name, db_path):
     mkdir_p(os.path.dirname(db_path))
 
     with open(db_path, "wb") as db_file:
-        pg_dump = "set -o pipefail; pg_dump -U dhis dhis2 | gzip"
+        pg_dump = "set -o pipefail; pg_dump -U dhis dhis2 --exclude-table 'analytics*' | gzip"
```

Comment:
To avoid repeating, can you create a function in utils.py that returns the array with the command (from pg_dump until then end) and use it here and run_sql.py?